### DISCUSSION TITLE
Input change event does not trigger with x-mask in Safari

### DISCUSSION THREAD

**Discussion by kg-currenxie** [alpinejs/alpine#4323] [7. Bugs (archived)]
Reproduced here: https://codesandbox.io/p/sandbox/alpine-js-forked-w7zdgq
Chrome works as expected

Steps:
1. Add a digit to the input (123) and blur. The change log is logged. ✅
2. Add another digit (1234) and blur. The input has been formatted with the mask, but the change event did not log anything ❌

**Comment by ekwoka** [comment]
<img width="153" alt="Screenshot 2024-07-29 at 11 32 15 AM" src="https://github.com/user-attachments/assets/b3341df5-f33a-4fac-b428-eead45bdf509">

Works on my machine (in your link)

**Reply by kg-currenxie** [reply]
Version? I'm at Version 17.5 (19618.2.12.11.6) - MacOS 14.5

**Reply by kg-currenxie** [reply]
https://github.com/user-attachments/assets/d2f92f4b-e9ef-4ee0-95cf-f24b0309cd04

**Reply by kg-currenxie** [reply]
We have 3 different people in our team with the same bug :|

**Reply by ekwoka** [reply]
oh i missed the `in Safari` :thinking:

Thats strange...

validated in Seqoia.
